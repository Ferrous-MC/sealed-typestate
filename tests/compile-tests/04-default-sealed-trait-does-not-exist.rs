use sealed_typestate::sealed_typestate;

sealed_typestate!(Foo<_> { Bar, Baz });

fn main() {}
