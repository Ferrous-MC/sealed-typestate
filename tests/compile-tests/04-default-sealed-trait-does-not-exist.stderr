error[E0433]: failed to resolve: could not find `private` in the crate root
 --> tests/compile-tests/04-default-sealed-trait-does-not-exist.rs:3:1
  |
3 | sealed_typestate!(Foo<_> { Bar, Baz });
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ could not find `private` in the crate root
  |
  = note: this error originates in the macro `sealed_typestate` (in Nightly builds, run with -Z macro-backtrace for more info)
